src/.env-example

```diff
@@ -1,2 +1,2 @@
 DJANGO_SECRET_KEY=""
-DJANGO_SECRET_LOGIN_KEY=""
+DJANGO_SECRET_LOGIN_KEY=""
```

Comment:
I would `git checkout filename` in your local branch + then push the commit up as there shouldn't be changes to this file (this probs happened on the call when we were copy pasta-ing from here for the Docker issue)! Otherwise looks great 🔥

Reply:
oof thank you for catching that! will remove this change :~)

Reply:
unblocked my GPG issues and reverted this change! will starting merging :)